docs/support.md

```diff
@@ -0,0 +1,9 @@
+# Getting Support
+
+For issues relating to documentation, facility deployments, and requests to include new packages in E4S, please file an issue here:
+* https://github.com/E4S-Project/e4s/issues
+
+For all other issues, please feel free to raise them in one of the following ways:
+1. [E4S Project Slack](https://e4s-project.slack.com)
+2. [E4S Channel on Spack Slack](https://spackpm.slack.com)
```

Comment:
perhaps rename it as `#e4s channel in spack slack`, the *#e4s* is more clear but i guess a question is whether E4S plans to monitor the spack slack #e4s and that can cause some confusion

Reply:
done